src/requirement_gatherer/tools.py

```
@@ -34,16 +32,21 @@
+        
+        If configured for human input, prompts the user and captures their response. If configured for AI, generates a reply using an AI model with specific behavioral instructions and updates the agent's state with the response.
+        
         Args:
-            question: The question to ask.
-
+            question: The question to present for feedback.
+        
         Returns:
-            A Command to update the state with the human response.
+            A Command that updates the agent's state with the feedback response.
         """
```

Ruff (0.8.2)

First line of docstring should be in imperative mood: "Requests feedback on a question from either a human user or an AI simulating a human, based on agent configuration."

(D401)